objects/_o-arrow.scss

```diff
@@ -6,6 +6,18 @@
 /**
  * Creates arrow shapes using only CSS.
  *
+ * All the objects and their children (available as silent placeholder selectors also):
```

Comment:
Looking at this again can we say—for all the objects—the following?

>> The objects block class, its modifier class(s), and its element class(s) (available as silent placeholder selectors also):

And wrap at 80 characters please.

Thanks